requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML>=3.10
-apispec==1.0.0b1
+apispec==1.3.3
```

Comment:
Could you please make this `<1.4`?

Reply:
Forgot that the `requirements.txt` is for setting up a dev env. When `pip install`ing, `setup.py` is used.

Could you also update this requirement in `setup.py`? Thanks!

Reply:
Oh good catch! I've made both the changes. Let me know if there's anything else!